Shared/Samples/Add rasters and feature tables from geopackage/README.md

```diff
@@ -0,0 +1,41 @@
+# Add rasters and feature tables from geopackage
+
+Add rasters and feature tables from a geopackage to a map.
+
+![Image of Add rasters and feature tables from geopackage sample](add-rasters-and-feature-tables-from-geopackage.png)
+
+## Use case
+
+The OGC GeoPackage specification defines an open standard for sharing raster and vector data. GeoPackages are designed to simplify file management and transfer. An end-user wishing to transfer rasters from ArcGIS Pro or between ArcGIS Maps SDK apps, might need to import raster files from GeoPackages into their map to view and analyze the data.
+
+## How to use the sample
+
+When the sample loads, the feature tables and rasters from the GeoPackage will be shown on the map.
+
+## How it works
+
+1. Open the GeoPackage using, specifying the path to the local .gpkg file.
```

Comment:
This sentence doesn't really make sense, is this better?

```suggestion
1. Open the GeoPackage using the path to the local .gpkg file.
```

Reply:
Oops great catch!